lib/rules/index.js

```diff
@@ -74,4 +74,5 @@ module.exports = {
   'style-concatenation': require('./style-concatenation'),
   'table-groups': require('./table-groups'),
   'template-length': require('./template-length'),
+  'no-redundant-fn': require('./no-redundant-fn'),
```

Comment:
We should sort these alphabetically, but I think @bmish mentioned doing a follow up to fix that (and the `yarn new` script).

Reply:
Done: https://github.com/ember-template-lint/ember-template-lint/pull/1455